js/hoot/control/conflicts/map/FeatureHighlighter.js

```diff
@@ -151,7 +151,8 @@ Hoot.control.conflicts.map.featurehighlighter = function (context)
         _parent().reviewIds.push(relId);
         _parent().info.metadata.updateMeta(null);
         if(panToId && panTo) {
-            _parent().map.featureNavigator.panToEntity(context.entity(panToId), true);
+            var bounds = feature.loc.toString() + ',' + againstFeature.loc.toString();
```

Comment:
Found the cause of the cuke error.  This won't work with features that are ways.  They don't have a "loc" property.